lib/kbsemi.gi

```diff
@@ -208,15 +209,25 @@ local r,kbrws,rwsfam,relations_with_correct_order,CantorList,relwco,
 
   # generates the list of all pairs (x,y)
   # where x,y are distinct elements of the set [1..n]
+  # encoded in compact form
   CantorList:=function(n)
-     local i,j,l;
+  local i,j,l;
+    l:=[];
+    for i in [2..n] do
+      Add(l,['A',i,[1..i-1]]);
+      Add(l,['B',[1..i-1],i]);
+    od;
+     return(l);
+
+
      l:=[];
      for i in [1..n] do
           Add(l,[i,i]);
           for j in [1..i-1] do
                Append(l,[[i,j],[j,i]]);
           od;
      od;
+Error();
```

Comment:
What is the point of this `Error` here? Is it just a leftover testing device from when you were writing the code?

Reply:
Actually the whole for loop before is dead as well -- the function has been rewritten completely and will never run into this old code. I'll remove it. Thanks.

Reply:
Oh yeah, I'd missed that there was already a `return` earlier in the function. Thanks